moq-loc/draft-mzanaty-moq-loc.md

```diff
@@ -284,6 +287,238 @@ with the header data as additional data input.
+                |   |    Tracks, Groups, Objects    |
+                |   +-------------------------------+
+ +--------------v---------------+
+ |     MOQT Object Model        |
+ +--------------+---------------+
+                |  +----------------------------------------+
+                |  |        One or more SubGroups           |
+                |  +----------------------------------------+
++---------------v--------------+
+|            QUIC              |
++------------------------------+
+~~~
+
+
+## Application with one audio track {#app-audio}
+
+Original publishers intending to publish audio, do so by advertising
+catalog with audio track information {{audioprofile}}.
+
+Below shows one such example for opus audio
+
+~~~psuedocode
+codec: "opus"
+bitrate: 24000
+samplerate: 480000
+channelConfig: "mono"
+lang: "en"
+~~~
+
+When ready for publishing, each chunk of encoded audio chunk, say 10ms, represents a MOQT Object. In this setup, there is one `MOQT Object`
+per `MOQT Group`, where the `GroupID` in the object header is
+increment by one for each encoded audio chunk and the `ObjectID`
+is defaulted to value 0. When mapped to the underlying QUIC Stream,
+each such unitary group is sent over individual unidirectional QUIC stream since there is just one `SubGroup` per each `MOQT Group`.
+
+
+## Application with one single quality video track {#app-1-video}
+
+The common properties of catalog describing a single video track with
+one spatial and temporal qualities are defined in {{profile}}.
+
+Here is one such example for a 720p, 30fps, h264 video track is
+as shown below:
+
+~~~psuedocode
+codec: "avc3.42E01E"
+bitrate: 1000000
+framerate: 30
+width: 1280
+height: 720
+~~~
+
+When ready for publishing, each encoded video chunk is considered as input
+to MOQT Object payload. If encrypted, the output of encryption will serve as
+the object's payload. The `GroupID` is incremented by 1 at IDR Frame boundaries. The `ObjectID` is increment by 1 for each encoded video frame, starting at 0 and resetting to 0 at the start of a new group. The first encoded video frame, MOQT Object with `ObjectID` 0, shall be the IDR Frame and the rest of the encoded video frames corresponds to dependent frames (delta frames) and organized in the decode order.
+
+When mapping to QUIC for sending, one unidirectional QUIC stream is setup to deliver all the encoded video chunks within a MOQT group.
+
+When decoding at the 'End Consumer', the objects from each of the QUIC
+streams are fed in the ObjectID order to the decoder setup for the
+video track.
+
+
+## Application with single video track with temporal layers {#app-2-temp-video}
+
+This section describes an media application with a single video track having
+2 temporal layers.
+
+Below shows sample catalog properties (see {{profile}} and {{video-ext}}) for describing H.264 video with 2 temporal qualities at 15fps and 30fps.
+
+~~~psuedocode
+
+Video Track 1: 720p 30/60fps H.264 video
```

Comment:
30/60 -> 30+60
Is there a catalog field to indicate 2 temporal layers? Seems missing here.